docs/community/developers.md

```diff
@@ -41,8 +41,8 @@ implementation):
 $ just build --no-default-features --features rustls-tls
 ```
 
-The same flags can be passed to `just run-wasi` and `just run-wascc` if you want
-to just [run](#running) the project instead.
+The same flags can be passed to `just run`. if you want to just [run](#running)
```

Comment:
looks like a period was added here by mistake?

Reply:
Yes it was